loops-arrays-objects.js

```diff
@@ -67,24 +83,31 @@ const NOVIEmployees = [
 // Bij NOVI horen de e-mailadressen altijd in het volgende format: voornaam.achternaam@novi.nl
 // Vóór het script zie je de originele objecten,
 // Na jouw script zie je de aangepaste objecten:
-// console.log(NOVIEmployees) geeft: [
+// console.log(NOVIEmployees) geeft:
+// [
 //   { firstName: 'Nova', lastName: 'Eeken', email: 'Nova.Eeken@novi.nl' },
 //   { firstName: 'Sam', lastName: 'Barnhoorn', email: 'Sam.Barnhoorn@novi.nl' },
 //   { firstName: 'Tessa', lastName: 'Steur', email: 'Tessa.Steur@novi.nl' },
 //   { firstName: 'Mark', lastName: 'Rensen', email: 'Mark.Rensen@novi.nl' },
-//  ];
-// ==========================================
-
-
+//  ];git status
 
+// ==========================================
 
+for (let i = 0; i < NOVIEmployees.length; i++) {
+    NOVIEmployees[i].email = NOVIEmployees[i].firstName + '.' + NOVIEmployees[i].lastName + '@novi.nl'
+    console.log(NOVIEmployees[i])
+}
+console.log(NOVIEmployees)
```

Comment:
Je slaat het email adres nu niet op. Je kan het email adres eventueel in een const verwerken.